safedata_validator/entry_points.py

```diff
@@ -594,6 +595,12 @@ def _safedata_zenodo_cli(args_list: list[str] | None = None) -> int:
         default=False,
         help="Should locally modified files be overwritten with the archive version",
     )
+    sync_local_dir_parser.add_argument(
+        "--dry-run",
+        action="store_true",
+        default=False,
+        help="Run the synchronisation process without altering the local directory.",
+    )
```

Comment:
This new option should be mentioned in the docs somewhere (probably not just api stuff, like a fuller explanation for when this option would be useful)